src/test/ui-fulldeps/pathless-extern-unstable.rs

```diff
@@ -0,0 +1,10 @@
+// ignore-stage1
+// edition:2018
+// compile-flags:--extern rustc
```

Comment:
What happens if we e.g. say `--extern alloc=$path_to_rustc_crate`... would that be gated as well?

Reply:
Yes, the gating mechanism is independent from crate locator/loading.

Reply:
Would be good to have tests exercising `--extern alloc=$path_to_rustc` being gated then.

Reply:
OK. It's not really practical to link to the real thing, but I added a test for a path to an unstable crate.

Reply:
Thanks